build.sbt

```diff
@@ -0,0 +1,35 @@
+
+name := "neo4j-scala-client"
+
+organization := "me.jeffmay"
+
+scalaVersion := "2.11.7"
+
+val playVersion = "2.4.6"
+val playJsonOpsVersion = "1.1.5"
+val scalatestVersion = "3.0.0-M10"
+
```

Comment:
HINT: Add the following scala options.

```
scalacOptions in ThisBuild := Seq(
  "-Xfatal-warnings",
  "-feature",
  "-deprecation:false",
  "-unchecked",
  "-Xfuture",
  "-Ywarn-unused-import"
)
```

HINT: consider adding a linter
```
libraryDependencies += compilerPlugin("org.psywerx.hairyfotr" %% "linter" % "0.1.12")
```

No better time than the first commit of a project